.github/workflows/build-and-test.yml

```diff
@@ -3,27 +3,27 @@ on:
   push:
     branches: [master]
     tags:
-      - v[0-9]+.[0-9]+.[0-9]+
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
   pull_request:
+env:
+  TEST_RESULTS: testbed/tests/results/junit/results.xml
 
 jobs:
   windows-test:
     runs-on: windows-latest
-    env:
-      TEST_RESULTS: testbed/tests/results/junit/results.xml
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
       - name: Setup Go
         uses: actions/setup-go@v2.1.3
         with:
           go-version: 1.15
-      - name: Setup env
+      - name: Setup Environment
         run: |
           echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
           echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
         shell: bash
-      - name: Restore module cache
+      - name: Cache Modules
```

Comment:
s/Cache Modules/Restore Cached Go Modules? Maybe a better suggestion? But I think it should include "Go Modules"

Reply:
I chose just cache over restore cache, because this not only restores the cache but also tells which path to save in cache after the job is complete. Renamed to "Cache Go Modules" as requested.